libosrmc/osrmc.cc

```diff
@@ -245,14 +256,40 @@ float osrmc_table_response_duration(osrmc_table_response_t response, unsigned lo
 
   auto& durations = response_typed->values["durations"].get<osrm::json::Array>();
   auto& durations_from_to_all = durations.values.at(from).get<osrm::json::Array>();
-  auto duration = durations_from_to_all.values.at(to).get<osrm::json::Number>().value;
+  auto optional = durations_from_to_all.values.at(to);
+
+  if (optional.is<osrm::json::Null>()) {
```

Comment:
For the record this would also resolve https://github.com/daniel-j-h/libosrmc/issues/1